subgraph/tests/metaBoard.test.ts

```
@@ -85,7 +89,7 @@ describe("Test meta event", () => {
 describe("Test MetaBoard and MetaV1 Entities", () => {
   beforeAll(() => {
     const meta = Bytes.fromHexString(metaString);
-    let newMetaV1Event = createNewMetaV1Event(sender, subject, meta);
+    let newMetaV1Event = createNewMetaV1Event(sender, subject, meta, transactionHash, transactionBlockNumber, transactionTimestamp);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newMetaV1Event' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)